### PR TITLE
chore(webpack): bump versions of webpack dependencies

### PR DESCRIPTION
**Pull request by taymoork2**
Hey there,
Thank you for taking the time to update our dependencies! 🙂

Fixes [SPARK-162648](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-162648)

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -136,7 +136,7 @@
     "grunt": "~1.0.1",
     "grunt-cli": "~1.2.0",
     "highlight.js": "^9.15.8",
-    "html-webpack-plugin": "^3.2.0",
+    "html-webpack-plugin": "^4.3.0",
```
Comment (taymoork2) [inline package.json:139]: I don't think we use this anywhere

Comment (adamweeks) [reply package.json:139]: It is used in the widgets and probably wasn't removed when we split the widgets out of the sdk. Feel free to remove it.

**Comment by adamweeks** [comment]
@taymoork2 did you verify that build works properly? We don't have any tests for build system changes, so we'll need to do a manual run and verify.

**Comment by taymoork2** [comment]
@adamweeks the build job should be enough of a indicator that updating webpack and it's dependencies is working
https://app.circleci.com/pipelines/github/webex/webex-js-sdk/3432/workflows/40abc091-69ab-48af-ab98-a2dec168cfe0/jobs/26276

running `NODE_ENV=production npm run build:script` locally and checking that the UMD build is minified with the correct version number indicates that nothing is broken as well

image

**Pull request merged**